validator/sawtooth_validator/networking/interconnect.py

```diff
@@ -309,10 +310,10 @@ def _dispatch_message(self):
                 else:
                     my_future = self._futures.get(message.correlation_id)
 
-                    LOGGER.debug("message round "
-                                 "trip: %s %s",
-                                 get_enum_name(message.message_type),
+                    LOGGER.debug("message round trip: %s %s",
```

Comment:
Seems way too granular, maybe better for metrics than logging?

Reply:
I can take that statement out, but anything more complicated is outside the scope of this PR.

Reply:
yeah, i vote for removing this and any other loggers that happen on every message

Reply:
That's a more extreme change. I'll do a separate PR for that in order to keep the resulting flame war from holding up this one.